.CurrentChangelog.md

```diff
@@ -1,24 +1,27 @@
-### 0.10.1-alpha.1 (2022-06-22)
+### 0.10.1-alpha.2 (2022-07-01)
 
 Features:
 
 * Supported WASI-NN plug-in with OpenVINO backend on Ubuntu 20.04 x86_64.
   * Users can refer to the [standard extension status](https://wasmedge.org/book/en/intro/standard.html) for the information.
+  * For building with enabling WASI-NN with OpenVINO backend, please add the `-DWASMEDGE_WASINN_BACKEND="OpenVINO"` in `cmake`.
 * Supported the WASM `threads` proposal.
   * Added the `WasmEdge_Proposal_Threads` for the configuration in WasmEdge C API.
   * Users can use the `--enable-threads` to enable the proposal in `wasmedge` and `wasmedgec` tools.
+* Supported LLVM 14 on MacOS.
+  * Used the new `macho` in lld on LLVM-14 envronment.
+  * Bumpped IWYU to 0.18 to be compatible with LLVM 14 on MacOS.
 
 Fixed issues:
 
+* Fixed the universal WASM format failed on MacOS platforms.
+  * Developers can specify the extension name as `.wasm` on MacOS as the universal WASM format output of the AOT compiler to enable the AOT mode.
 * Fixed the WasmEdge C API static library on MacOS with LLVM 14.
   * The WasmEdge C API static library is in experimental and not guaranteed. The shared library is recommended.
+* Fixed the branch miss when instantiating AOT-compiled WASM.
```

Comment:
This should be in the improvement or changes. And it should start with `Reduced the branch miss...`.